src/services/user.js

```diff
@@ -390,17 +390,36 @@ const getLocalInfo = () => {
   }
 };
 
+function sortAtndEventsByTime(a, b) {
+
+  let timeA = a.CHDate;
```

Comment:
Good job finding a different property to sort on. It might be helpful to leave a comment here about the meaning and relative accuracy of CHDate versus Occurrences[0][0], so that some future maintainer is not mystified by this code.

Reply:
Good point.